tests/typescript/as/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -191,6 +202,17 @@ this.intervalID = (setInterval(() => {
 ================================================================================
 `;
 
+exports[`assignment2.ts [babel-ts] format 1`] = `
+"Invalid parenthesized assignment pattern (5:1)
+  3 | ) => Mask;
+  4 |
+> 5 | (this.configuration as any) = (this.editor as any) = (this
+    | ^
+  6 |   .editorBody as any) = undefined;
```

Comment:
This is not right.

Reply:
Weird. Babel supports this since long ago: https://github.com/babel/babel/pull/7098
Why do we get an error?

Reply:
Turns out this is because of `createParenthesizedExpressions: true`.

Reply:
Opened https://github.com/babel/babel/issues/12932

Reply:
Disabling `createParenthesizedExpressions` for `babel-ts` is a solution.